bundlesize.config.json

```diff
@@ -1,8 +1,8 @@
 {
   "files": [
     {
-      "path": "visual-testing-app/dist/ui-kit.js",
-      "maxSize": "190 kB"
+      "path": "packages/**/dist/*.es.js",
+      "maxSize": "60 kB"
```

Comment:
I'm not sure why we are checking the bundle in the visual testing app. We should check each package dist.

```
yarn bundlesize
yarn run v1.22.10
$ bundlesize
 PASS  packages/calendar-time-utils/dist/calendar-time-utils.es.js: 1.52KB < maxSize 60KB (gzip) 

 PASS  packages/calendar-utils/dist/calendar-utils.es.js: 1.39KB < maxSize 60KB (gzip) 

 PASS  packages/components/accessible-hidden/dist/accessible-hidden.es.js: 490B < maxSize 60KB (gzip) 

 PASS  packages/components/avatar/dist/avatar.es.js: 1.74KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/accessible-button/dist/accessible-button.es.js: 8.81KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/flat-button/dist/flat-button.es.js: 2KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/icon-button/dist/icon-button.es.js: 5.73KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/link-button/dist/link-button.es.js: 8.19KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/primary-button/dist/primary-button.es.js: 2.38KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/secondary-button/dist/secondary-button.es.js: 3.21KB < maxSize 60KB (gzip) 

 PASS  packages/components/buttons/secondary-icon-button/dist/secondary-icon-button.es.js: 1.67KB < maxSize 60KB (gzip) 

 PASS  packages/components/card/dist/card.es.js: 885B < maxSize 60KB (gzip) 

 PASS  packages/components/collapsible-motion/dist/collapsible-motion.es.js: 2.47KB < maxSize 60KB (gzip) 

 PASS  packages/components/collapsible-panel/dist/collapsible-panel.es.js: 10.48KB < maxSize 60KB (gzip) 

 PASS  packages/components/collapsible/dist/collapsible.es.js: 1.08KB < maxSize 60KB (gzip) 

 PASS  packages/components/constraints/dist/constraints.es.js: 3.46KB < maxSize 60KB (gzip) 

 PASS  packages/components/data-table-manager/dist/data-table-manager.es.js: 15.14KB < maxSize 60KB (gzip) 

 PASS  packages/components/data-table/dist/data-table.es.js: 16.64KB < maxSize 60KB (gzip) 

 PASS  packages/components/field-errors/dist/field-errors.es.js: 1.55KB < maxSize 60KB (gzip) 

 PASS  packages/components/field-label/dist/field-label.es.js: 1.11KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/async-creatable-select-field/dist/async-creatable-select-field.es.js: 4.57KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/async-select-field/dist/async-select-field.es.js: 4.13KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/creatable-select-field/dist/creatable-select-field.es.js: 4.29KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/date-field/dist/date-field.es.js: 2.75KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/date-range-field/dist/date-range-field.es.js: 2.73KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/date-time-field/dist/date-time-field.es.js: 2.74KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/localized-multiline-text-field/dist/localized-multiline-text-field.es.js: 3.35KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/localized-text-field/dist/localized-text-field.es.js: 3.27KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/money-field/dist/money-field.es.js: 9.98KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/multiline-text-field/dist/multiline-text-field.es.js: 2.75KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/number-field/dist/number-field.es.js: 2.82KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/password-field/dist/password-field.es.js: 2.38KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/radio-field/dist/radio-field.es.js: 2.76KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/search-select-field/dist/search-select-field.es.js: 3.43KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/select-field/dist/select-field.es.js: 3.93KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/text-field/dist/text-field.es.js: 2.67KB < maxSize 60KB (gzip) 

 PASS  packages/components/fields/time-field/dist/time-field.es.js: 2.66KB < maxSize 60KB (gzip) 

 PASS  packages/components/grid/dist/grid.es.js: 7.83KB < maxSize 60KB (gzip) 

 PASS  packages/components/icons/dist/icons.es.js: 31.12KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/async-creatable-select-input/dist/async-creatable-select-input.es.js: 6.45KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/async-select-input/dist/async-select-input.es.js: 6.12KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/checkbox-input/dist/checkbox-input.es.js: 11.21KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/creatable-select-input/dist/creatable-select-input.es.js: 6.35KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/date-input/dist/date-input.es.js: 14.7KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/date-range-input/dist/date-range-input.es.js: 16.06KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/date-time-input/dist/date-time-input.es.js: 16.04KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/localized-money-input/dist/localized-money-input.es.js: 3.98KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/localized-multiline-text-input/dist/localized-multiline-text-input.es.js: 13.85KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/localized-rich-text-input/dist/localized-rich-text-input.es.js: 53.31KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/localized-text-input/dist/localized-text-input.es.js: 4.62KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/money-input/dist/money-input.es.js: 25.14KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/multiline-text-input/dist/multiline-text-input.es.js: 3.65KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/number-input/dist/number-input.es.js: 2.55KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/password-input/dist/password-input.es.js: 2.29KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/radio-input/dist/radio-input.es.js: 10.5KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/rich-text-input/dist/rich-text-input.es.js: 48.27KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/search-select-input/dist/search-select-input.es.js: 9.73KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/select-input/dist/select-input.es.js: 6.5KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/select-utils/dist/select-utils.es.js: 2.25KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/text-input/dist/text-input.es.js: 2.21KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/time-input/dist/time-input.es.js: 9.55KB < maxSize 60KB (gzip) 

 PASS  packages/components/inputs/toggle-input/dist/toggle-input.es.js: 8.31KB < maxSize 60KB (gzip) 

 PASS  packages/components/label/dist/label.es.js: 969B < maxSize 60KB (gzip) 

 PASS  packages/components/link/dist/link.es.js: 1.41KB < maxSize 60KB (gzip) 

 PASS  packages/components/loading-spinner/dist/loading-spinner.es.js: 1.17KB < maxSize 60KB (gzip) 

 PASS  packages/components/messages/dist/messages.es.js: 647B < maxSize 60KB (gzip) 

 PASS  packages/components/notifications/dist/notifications.es.js: 1.69KB < maxSize 60KB (gzip) 

 PASS  packages/components/primary-action-dropdown/dist/primary-action-dropdown.es.js: 9.24KB < maxSize 60KB (gzip) 

 PASS  packages/components/spacings/inline/dist/inline.es.js: 999B < maxSize 60KB (gzip) 

 PASS  packages/components/spacings/inset-squish/dist/inset-squish.es.js: 706B < maxSize 60KB (gzip) 

 PASS  packages/components/spacings/inset/dist/inset.es.js: 616B < maxSize 60KB (gzip) 

 PASS  packages/components/spacings/stack/dist/stack.es.js: 952B < maxSize 60KB (gzip) 

 PASS  packages/components/stamp/dist/stamp.es.js: 1.32KB < maxSize 60KB (gzip) 

 PASS  packages/components/table/dist/table.es.js: 15.38KB < maxSize 60KB (gzip) 

 PASS  packages/components/tag/dist/tag.es.js: 9.01KB < maxSize 60KB (gzip) 

 PASS  packages/components/text/dist/text.es.js: 2.93KB < maxSize 60KB (gzip) 

 PASS  packages/components/tooltip/dist/tooltip.es.js: 10.55KB < maxSize 60KB (gzip) 

 PASS  packages/hooks/dist/hooks.es.js: 2.18KB < maxSize 60KB (gzip) 

 PASS  packages/i18n/dist/i18n.es.js: 4.98KB < maxSize 60KB (gzip) 

 PASS  packages/localized-utils/dist/localized-utils.es.js: 1.78KB < maxSize 60KB (gzip) 

 PASS  packages/utils/dist/utils.es.js: 2.26KB < maxSize 60KB (gzip) 
```